src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -1233,6 +1239,22 @@ void StartImportingInstruction()
 
         void EndImportingInstruction()
         {
+            var currentOpcode = (ILOpcode)_ilBytes[_currentInstructionOffset];
```

Comment:
Do we need to handle situation where we are at the end of the instruction stream here (_currentInstructionOffset >= _ilBytes.Length)?

Reply:
I think it shouldn't be possible that `EndImportingInstruction` is called with an invalid `_currentInstructionOffset`.
`_currentInstructionOffset` is set in `StartImportingInstruction`, then the opCode is parsed immediately after that. If the instruction offset is invalid at that point we should already crash there.